Phonebook.Frontend/src/app/modules/table/components/table/table.component.ts

```diff
@@ -92,6 +77,36 @@ export class TableComponent implements OnInit, OnDestroy {
     });
   }
 
+  public loadPersonData(ignoreCache: boolean = false) {
+    this.inErrorState = false;
+    this.personService.getAll(ignoreCache).subscribe(
+      (persons) => {
+        this.dataSource = new PersonsDataSource(persons);
+
+        // Defer until Data is loaded.
+        this.refreshTableSubscription = merge(
+          this.store.select(SearchState.searchTerm),
+          this.store.select(SearchState.searchFilters)
+        )
+          .pipe(
+            // Debounce Time is this low, as it just aims to bundle all Observables above, especially at first page load,
+            // where all three fire as they are initialized.
+            debounceTime(50)
+          )
+          .subscribe((val) => {
+            this.refreshTable();
+            this.dataSource.pageSize = this.initialPageSize;
+          });
+      },
+      (err: HttpErrorResponse) => {
+        // All 5xx errors
```

Comment:
This does not catch offline errors. I think we should display the error message indifferent of the error response.